prisma/schema.prisma

```diff
@@ -170,6 +171,9 @@ enum EventType {
   SOCIAL_MEDIA_PROMOTION
   NODE_UPTIME
   SEND_TRANSACTION
+  MASP_TRANSFER
+  MASP_BURN
+  MASP_MINT
```

Comment:
Enum previously didn't have entries for event type

Reply:
How are these events going to be categorized? Will there be a single MASP type? I'm wondering if we should have types here and remove the type from the `masp_transaction` table.

Reply:
Seems reasonable to me, I can remove from the `masp_transaction` table